build.sbt

```diff
@@ -43,6 +43,12 @@ inThisBuild {
     testFrameworks += new TestFramework("munit.Framework"),
     // causes native image issues
     dependencyOverrides += "org.jline" % "jline" % "3.29.0",
+    publishTo := {
```

Comment:
I think you don't need to set this when using sbt-ci-release, the plugin sets it by default.

Reply:
Thanks! Removed